src/core/errorHandler.js

```diff
@@ -59,13 +59,8 @@ export class ErrorHandler {
 export type ErrorHandlerType = typeof ErrorHandler;
 
 /*
- * This is a decorator that gives a component the ability to handle errors.
- *
- * The decorator will assign an ErrorHandler instance to the errorHandler
- * property.
```

Comment:
Doh! @tofumatt [suggested to shorten the comment](https://github.com/mozilla/addons-frontend/pull/2944#discussion_r133213126), so I thought I could get rid of these lines.